### PR TITLE
Improve windows code signing documentation

### LINKED ISSUE #5066: Document code signing support for Windows Desktop exports
**Your Godot version:**
3.4.beta.custom_build.8aa28f208

**Issue description:**
The following pull request:

* https://github.com/godotengine/godot/pull/32556

added code signing support for the Windows Desktop platform into the engine, but it remains undocumented.

There is no mention of osslsigncode in the user documentation.

**URL to the documentation page:**
Perhaps the current documentation for the Universal Windows Platform can be leveraged to write the new documentation:
https://docs.godotengine.org/en/stable/getting_started/workflow/export/exporting_for_uwp.html

### PR DESCRIPTION
**Pull request by skyace65**
The windows code signing documentation was extremely bare bones, this expands it and explains what settings need to be set. Closes #5066. Both images have gone through png compression.

### CODE REVIEW

File: tutorials/export/exporting_for_windows.rst [hunk 1]
```diff
@@ -27,3 +29,23 @@ Requirements
...
     you're distributing your project via Steam as it bypasses code signing and
     antivirus checks.
+
+Setup
+~~~~~
+
+Settings need to be changed in two places. First, in the editor settings, under
+**Export > Windows**. Click on the folder next to the ``Sign Tool`` setting, if
+you're using Windows navigate to and select SignTool.exe, if you're on a different
+OS select osslsigncode.
```
Comment (mhilbrunner) [inline tutorials/export/exporting_for_windows.rst:39]:
```suggestion
you're using Windows navigate to and select ``SignTool.exe``, if you're on a different
OS select ``osslsigncode``.
```

**Review by mhilbrunner — APPROVED** [review]
Small formatting quibble, looks good to go otherwise

**Comment by skyace65** [comment]
Fixed.

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thanks!

**Pull request merged**